test/functional/terminal/ex_terminal_spec.lua

```diff
@@ -112,4 +112,46 @@ describe(':terminal (with fake shell)', function()
       eq(2, eval("1+1"))  -- Still alive?
   end)
 
+  describe('works with', function()
+    it('findfile()', function()
+      source('terminal')
+      eq(1, nvim('eval', 'bufname("%") =~# "^term://"'))
```

Comment:
comparing the actual value makes failures more informative:

    eq('^term://', nvim('eval', 'bufname("%")'))